README.rst

```diff
@@ -36,7 +36,7 @@ its members and the User Community to get the most out of Project ACRN.
 
 Welcome to the project ARCN community!
 
-We're now holding weekly Technical Community Meetings and encourage you
+We're now holding monthly Technical Community Meetings and encourage you
 to call in and learn more about the project. Meeting information is on
 the `TCM Meeting page`_ in our `ACRN wiki <https://wiki.projectacrn.org/>`_.
```

Comment:
"TCM" appears to mean "Technical Community Meeting", so "TCM Meeting" is redundant. "TCM Meeting page" → "TCM page", and headings on the wiki such as "ACRN TCM Meeting Minutes" should be changed as well (e.g., to "ACRN TCM Minutes").

Reply:
I think there's precedent for repeating the last acronym letter's word, as in "IRS service center", "TWA airlines", "PIN number", "UPC code", "IP protocol", and even the double "CSS style sheet". I think it reads better repeating "meeting".

Reply:
That's a valid point. Actually, every single one of the precedents you cited bugs me (along with "ATM machine"); however, we can just categorize this as my own personal problem. :-)